lms/envs/test.py

```diff
@@ -594,3 +594,6 @@
 ############### Settings for Django Rate limit #####################
 
 RATELIMIT_RATE = '2/m'
+
+# HOTJAR ANALYTICS
+HOTJAR_TRACKING_ID = None
```

Comment:
I dont think we need it here.

Reply:
Not adding it here breaks the code and starts to give errors. Debugged this last time with @taimoor-ahmed-1